Program.cs

```diff
@@ -1,7 +1,13 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+Console.Write ("Enter input: ");
+int n = int.Parse (Console.ReadLine ());
+int a = 0, b = 1, c, i = 2;
+Console.Write ($"{a} ");
+Console.Write ($"{b} ");
+while (n > i) {

```

Comment:
You can use the Post-Increment (i.e. i++) in the expression itself. Can you try and update?